openshift-knative-operator/cmd/operator/kodata/knative-eventing/1.11/1-eventing-crds.yaml

```diff
@@ -953,7 +953,7 @@ spec:
     schema:
```

Comment:
We probably want Eventing in first? As there seem to be differences already? @creydr @pierDipi 

/hold